tests/cli/net_handler_cli.c

```diff
@@ -315,6 +315,35 @@ void prompt_reroute_output() {
     update_tcp_output_fp(dest);
 }
 
+void print_next_dev_data() {
```

Comment:
Can you explain why you decided to move this function to the net_handler_cli? Personally, I always viewed the cli as  an area that displays values or takes in inputs, rather than getting the data and parsing it. I feel the majority of this should be within the client

Reply:
Talked about this during a meeting some time ago.